# Force-hide ability build suggestions in Street Brawl mode.
#
# Higher specificity (0,3,0) overrides always-show-ability-suggestions mod (0,2,0) and vanilla rules.
--- a/panorama/styles/hud_ability_icon.css
+++ b/panorama/styles/hud_ability_icon.css
@@ -1722,3 +1722,8 @@
 	font-family: block;
 }

+
+.gamemode_streetbrawl .build_suggestion .build_suggestion_container
+{
+	visibility: collapse;
+}
